### PR TITLE
docs(core): add documentation for core code

### PR DESCRIPTION
**Pull request by RaghavaroraSF**
## Description
docs(core): add documentation for core code

Added documentation for the code written in core

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts [hunk 1]
```diff
@@ -29,6 +38,11 @@ export class FacadesBearerTokenVerifyProvider
     public authUserModel?: Constructor<EntityWithIdentifier & IAuthUser>,
   ) {}
 
+  /**
+   *  @param VerifyFunction.BearerFn fucntion that will be called when the token is verified.
```
Comment (yeshamavani) [inline packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts:42]: explain in detail what this function actually does

File: packages/core/src/components/bearer-verifier/repositories/revoked-token.repository.ts [hunk 2]
```diff
@@ -8,6 +8,11 @@ import {DefaultKeyValueRepository, juggler} from '@loopback/repository';
 import {RevokedToken} from '../models';
 import {AuthCacheSourceName} from '../types';
 
+/**
+ * `RevokedTokenRepository` a class that extends the `DefaultKeyValueRepository` class,
+ *  which is a class that comes with LoopBack 4,
+ *  uses the AuthCacheSource data source to store and retrieve data.
```
Comment (yeshamavani) [inline packages/core/src/components/bearer-verifier/repositories/revoked-token.repository.ts:14]: AuthCache stores tokens only

Comment (RaghavaroraSF) [reply packages/core/src/components/bearer-verifier/repositories/revoked-token.repository.ts]: done

File: packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts (hunk 1 above)
Comment (RaghavaroraSF) [reply packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts]: done

**Comment by shubhamp-sf** [comment]
Closing as the changes isn't what was expected and It just rewrites technical bits of the classes/props etc. Any superseding PRs for this will need jsdoc formatted comments in easy language for just the "Publicly exported Classes or Mixins excluding the controllers or repositories".

**Pull request closed without merging**